.github/workflows/build_wheels_linux.yml

```diff
@@ -141,23 +142,31 @@ jobs:
           if [[ "${{ inputs.architecture }}" = "aarch64" ]]; then
             rm -rf "${RUNNER_TEMP}/*"
           fi
-
           echo "::endgroup::"
+
       - uses: actions/checkout@v4
+        if: ${{ env.IS_MANYLINUX2_28 == 'true' }}
         with:
           # Support the use case where we need to checkout someone's fork
           repository: ${{ inputs.test-infra-repository }}
           ref: ${{ inputs.test-infra-ref }}
           path: test-infra
-      - name: Set linux aarch64 CI
+
+      - uses: atalman/checkout-action@main
+        if:  ${{ env.IS_MANYLINUX2_28 != 'true' }}
+        with:
+          repository: ${{ inputs.test-infra-repository }}
+          ref: ${{ inputs.test-infra-ref }}
+          path: test-infra
```

Comment:
If you want to replace it with script, let's used it everywhere...

Reply:
Yes maybe its a good idea, we should be able to catch any possible issues with this script faster